src/main/java/com/example/library/model/Book.java

```diff
@@ -21,6 +23,8 @@ public class Book {
 
   @ManyToOne
   @JoinColumn(name = "author_id")
+  @EqualsAndHashCode.Exclude
+  @ToStringExclude
```

Comment:
Is this annotation actually picked up by the Lombok annotation preprocessor?
I would say that you need a different annotation here, like
```
import lombok.ToString.Exclude;
```

And then adding the following:
```
  @Exclude
  private Author author;
```